### ISSUE TITLE
'Found developers' shows up even when the user has not searched yet / when the search input is blank

### ISSUE THREAD

**Issue by Dhaiwat10** [moonshotcollective/drecruit#62]
![image](https://user-images.githubusercontent.com/39617427/141756849-9cb49463-eb39-4a26-91c4-829d7177827a.png)

**Comment by QEDK** [comment]
In my opinion, this does not need to be specified at all, search UIs are well-researched, it should just show when searched... and when the textbox is cleared, we can show all.

**Comment by Cali93** [comment]
Yeah I think that it might be better to use a state management library to handle the search & filters.
I'll work on a version with easy-peasy https://easy-peasy.vercel.app/docs/tutorials/quick-start.html
